airflow/providers/microsoft/azure/hooks/wasb.py

```diff
@@ -291,9 +329,13 @@ def upload(
             ``PageBlob`` or ``AppendBlob``. The default value is ``BlockBlob``.
         :param length: Number of bytes to read from the stream. This is optional,
             but should be supplied for optimal performance.
+        :param try_container_create: Flag to attempt to create the target container prior to uploading the
+            blob. This is useful if the target container may not be known to exist. Defaults to False.
```

Comment:
```suggestion
        :param try_container_create: Attempt to create the target container prior to uploading the
            blob. This is useful if the target container may not exist yet. Defaults to False.
```

The `try` part keeps leading me to Python’s `try` which does not really fit the semantics here. But I don’t have a very good recommendation either.

Reply:
Same. I initially had `attempt_container_creation` and I thought it was fluffy. Couldn't really come up with anything better really. But maybe that is better?

Reply:
Is `create_container_if_needed` much too long? Python actually generally prefers much more succinct names such as `container=True` (see `pathlib.Path.mkdir`) but we probably don’t need to go that far. Maybe `create_container=True`?

Reply:
>Is create_container_if_needed much too long?
I think so. Seems like a LBYL action. I do like `create_container` it's exactly what's configurable now. I'll update. Not sure why that escaped me before.